### PR TITLE
#21 - refactor create project class

### PR DESCRIPTION
**Pull request by mgibbs1259**
## Pull Request for Issue #21 

**Description of Changes**
We refactored CreateProject(). In testutils, we created an actions.py file that includes methods from CreateProject(). We updated the tests in create_project.py to call the methods from actions.py in testutils.

**Exit Criteria**
Tested all test methods in create_project.py

### CODE REVIEW

File: create_project.py [hunk 1]
```diff
@@ -22,280 +21,173 @@
...
+
+    Args:
+        driver(selenium.webdriver): The webdriver.
+
+    Returns:
+        driver(selenium.webdriver): The webdriver.
+    """
+    # project set up
+    testutils.log_in(driver)
+    time.sleep(2)
```
Comment (billvb) [inline create_project.py]: Just curious, what if this sleep isn't here? Does the test still fail?

File: testutils/actions.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+from testutils import testutils
+
+
+# create project
+def log_in(driver):
+    """
+    Log in to Gigantum.
+
+    Args:
+    driver(selenium.webdriver): The webdriver.
```
Comment (billvb) [inline testutils/actions.py]: you can make the signature `def log_in(driver: selenium.webdriver):` and not put the type in the comments

File: testutils/actions.py [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        pass
+    while not r_base_elts.r_tidyverse_base_button.is_displayed():
+        logging.info("Searching for R Tidyverse base...")
+        r_base_elts.arrow_button.click()
+    r_base_elts.r_tidyverse_base_button.click()
+    r_base_elts.create_project_button.click()
+
+
+# environment
+def add_pip_package(driver):
```
Comment (billvb) [inline testutils/actions.py]: We can paramterize this function with the list of packages, but that can be later

File: testutils/actions.py [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+    Returns:
+        driver(selenium.webdriver): The webdriver.
+    """
+    logging.info("Adding valid custom Docker")
+    environment = elements.EnvironmentElements(driver)
+    environment.environment_tab_button.click()
+    driver.execute_script("window.scrollBy(0, 600);")
+    environment.custom_docker_edit_button.click()
+    time.sleep(2)
```
Comment (billvb) [inline testutils/actions.py]: I wonder if these sleeps can go away

File: create_project.py (hunk 1 above)
Comment (dvitale199) [reply create_project.py]: Yes, this test fails without the sleep

File: testutils/actions.py (hunk 4 above)
Comment (mgibbs1259) [reply testutils/actions.py]: I removed these time sleeps, and it works fine for me.

**Pull request merged**